assets/javascripts/discourse/controllers/admin-wizards-submissions-show.js.es6

```diff
@@ -24,6 +27,21 @@ export default Controller.extend({
         this.set("loadingMore", false);
       });
   },
+  
+  
+  @discourseComputed('submissions', 'fields.@each.enabled')
+  displaySubmissions(submissions, fields) {
+    return submissions.map(submission => {
+      let field = fields.find(f => Object.keys(submission).includes(f.id));
+      if (!field.enabled) {
+        // insert field / submission deletion code here:
```

Comment:
Here is where I am having some issues, as discussed in our PM.

Reply:
Perhaps we could use a reduce here?